slices/libcurl4.yaml

```diff
@@ -0,0 +1,21 @@
+# Easy-to-use client-side URL transfer library (OpenSSL flavour)
+
```

Comment:
nit: let's get rid of this blank line too so that comment is directly above the `package` field.

Reply:
you are right @rebornplusplus  libnghttp was missed, now added, PR is updated with fixes, kindly review.